internal/cache/cache.go

```diff
@@ -81,7 +81,7 @@ func (c *Cache) fileName(id [HashSize]byte, key string) string {
 var errMissing = errors.New("cache entry not found")
 
 func IsErrMissing(err error) bool {
-	return err == errMissing
+	return errors.Cause(err) == errMissing
```

Comment:
Shouldnt this be https://godoc.org/github.com/pkg/errors#Is

Reply:
thank you, you are right. We will fix when will be replacing `pkg/errors` by `errors`. I guess it can happen with go1.15.